sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosChangeFeedRequestOptions.java

```diff
@@ -402,7 +403,47 @@ public CosmosChangeFeedRequestOptions fullFidelity() {
             );
         }
 
-        this.mode = ChangeFeedMode.FULL_FIDELITY;
+        this.mode = ChangeFeedMode.ALL_VERSIONS_AND_DELETES;
+        return this;
+    }
+
+    /**
+     * Changes the change feed mode so that the change feed will contain events for creations,
+     * deletes as well as all intermediary snapshots for updates. Enabling {@link ChangeFeedMode#ALL_VERSIONS_AND_DELETES}
+     * change feed mode requires configuring a retention duration in the change feed policy of the
+     * container. {@link ChangeFeedPolicy}
+     * <p>
+     * Intermediary snapshots of changes as well as deleted documents would be
+     * available for processing for 8 minutes before they vanish.
```

Comment:
where the 8 minutes comes from? it is a different than the retention window?

Reply:
I think it is just a example we added in the documentation. I will fix it.